### ISSUE TITLE
Can't update assemblies.

### ISSUE THREAD

**Issue by Maelep** [codemerx/CodemerxDecompile#73]
I have decompiled some DLL that I use everyday.
Yesterday, an update of these DLL has been released so I downloaded them and replaced the old one.
The decompiler is still loading the "cached" version, even if I clear the list (File > Clear assembly) and reload them.
The only way to load the new files is by changing the location of my files.

I also tried to load files in the new location, clear assembly and load again at the old location, without success.
Restarting my computer still doesn't resolve this issue.

**Comment by atdimitrov** [comment]
Thank you for letting us know about this issue. Would it be possible that you provide us with the 2 versions of the assembly that are causing the issue?

As a workaround:
1. Close CodemerxDecompile
2. Delete `%APPDATA%\CodemerxDecompile\sources`
3. Reopen CodemerxDecompile and load the assembly

**Comment by Maelep** [comment]
I discovered this bug when updating Oxide from [2.0.5164](https://github.com/OxideMod/Oxide.Rust/releases/download/2.0.5164/Oxide.Rust.zip) to [2.0.5186](https://github.com/OxideMod/Oxide.Rust/releases/download/2.0.5186/Oxide.Rust.zip).

Thank you for the workaround because changing file location at each update is boring xD

**Comment by atdimitrov** [comment]
Thank you for the follow up and for the reproduction scenario. We really appreciate that.

> Thank you for the workaround because changing file location at each update is boring xD

Sure, no problem. I thought we had this `Clear assembly list` functionality done, but it seems it is not fully functional yet, so a workaround is must here :)

**Comment by Maelep** [comment]
Do you think possible to create a small DOS script that:
- Clear cache.
- Load Assembly again.

I know how to clear cache in DOS but I don't know if it's possible to make CodemerxDecompile loads the assembly.

**Comment by atdimitrov** [comment]
No, I don't think that's possible. CodemerxDecompile does not support command line arguments as of yet.
